src/coreclr/jit/gentree.cpp

```diff
@@ -21017,6 +21017,22 @@ GenTree* Compiler::gtNewSimdBinOpNode(
 
             return gtNewSimdBinOpNode(GT_AND, type, shiftOp, maskOp, simdBaseJitType, simdSize);
         }
+
+        case GT_LSH:
+        {
+            assert(varTypeIsByte(simdBaseType));
+            unsigned shiftCountMask = (genTypeSize(simdBaseType) * 8) - 1;
+            GenTree* andShiftVal =
+                gtNewOperNode(GT_AND, op2ForLookup->TypeGet(), op2ForLookup, gtNewIconNode(shiftCountMask));
```

Comment:
This should already be getting done a few lines up: https://github.com/dotnet/runtime/pull/108336/files#diff-fba867eda1c745875748370c59da2195c3823b85a17d199444068d93f91c749bL20796-L20855